color/p5js/Motion-Figure:Ground/index.md

```diff
@@ -0,0 +1,9 @@
+layout: p5js
+author: Edmundo Mejía Galindo
+author-url: https://github.com/edgalindo/
+title: Law of Multistable-perception, Law of Relativity of Properties
+tags: [P5.js, Law of Multistable-perception, Law of Relativity of Properties, Law of Emotion, data, form, grid, motion]
+permalink: /Motion-Figure/Ground/
```

Comment:
lowercase. No categories given. Please don't create random permalinks. The peramlink should be:  

a category + the name of the sketch